WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
If it is just for the name we can solve this through other means. Because I currently don't understand  why this change is made, and the title isn't strictly secular.